From petkovbb@googlemail.com  Sun Feb  8 13:33:08 2009
From: Borislav Petkov <petkovbb@googlemail.com>
Date: Sat, 7 Feb 2009 20:26:31 +0100
Subject: USB: usb_storage: make Kconfig note visible in the console
To: mdharm-usb@one-eyed-alien.net
Message-ID: <20090207192631.GA21549@gollum.tnic>


From: Borislav Petkov <petkovbb@gmail.com>
Date: Sat, 7 Feb 2009 20:20:42 +0100
Subject: [PATCH] usb_storage: make Kconfig note visible in the console

Make lines about usb_storage depending on SCSI visible when configuring the
kernel in a 80x25 console

Signed-off-by: Borislav Petkov <petkovbb@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/storage/Kconfig |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/storage/Kconfig
+++ b/drivers/usb/storage/Kconfig
@@ -2,8 +2,8 @@
 # USB Storage driver configuration
 #
 
-comment "NOTE: USB_STORAGE depends on SCSI but BLK_DEV_SD may also be needed;"
-comment "see USB_STORAGE Help for more information"
+comment "NOTE: USB_STORAGE depends on SCSI but BLK_DEV_SD may"
+comment "also be needed; see USB_STORAGE Help for more info"
 	depends on USB
 
 config USB_STORAGE
